fe/fe-core/src/main/java/com/starrocks/sql/ast/CreateRoutineLoadStmt.java

```diff
@@ -136,7 +136,7 @@ public class CreateRoutineLoadStmt extends DdlStmt {
     public static final String PULSAR_DEFAULT_INITIAL_POSITION = "pulsar_default_initial_position";
 
     private static final String NAME_TYPE = "ROUTINE LOAD NAME";
-    private static final String ENDPOINT_REGEX = "[-A-Za-z0-9+&@#/%?=~_|!:,.;]+[-A-Za-z0-9+&@#/%=~_|]";
+    private static final String ENDPOINT_REGEX = "[-A-Za-z0-9+&@#/%?=~_|!:,.;\\[\\]]+[-A-Za-z0-9+&@#/%=~_|]";
```

Comment:
How about using a stricter regular expression to match IPv6 address?

Reply:
Can you provide an example? I think more strict may lead to more complexity.

Reply:
Variable name "IPV6_REGEX" is better?

Reply:
Exactly. I updated the current PR, please review again.